ignite/contrib/metrics/regression/canberra_metric.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import division
+import torch
+from ignite.metrics.metric import Metric
+
+
+class CanberraMetric(Metric):
+    r"""
+    Calculates the Canberra Metric.
```

Comment:
Could you please unify the docstring like for other implementations, e.g
```
    Calculates the Manhattan Distance:
    :math:`\text{MD} = \sum_{j=1}^n (A_j - P_j)`,
    where :math:`A_j` is the ground truth and :math:`P_j` is the predicted value.
    More details can be found in `Botchkarev 2018`__.
    - `update` must receive output of the form `(y_pred, y)`.
    - `y` and `y_pred` must be of same shape.
    __ https://arxiv.org/abs/1809.03006

```

Reply:
Done!